core/audits/byte-efficiency/legacy-javascript.js

```diff
@@ -170,105 +176,29 @@ class LegacyJavascript extends ByteEfficiencyAudit {
       // TODO: perhaps this is the wrong place to check for a CDN polyfill. Remove?
       // expression += `|;e\\([^,]+,${qt(objectWithoutPrototype)},{${property}:`;
 
-      // core-js@2 minified pattern.
-      // $export($export.S,"Date",{now:function
-      expression += `|\\$export\\([^,]+,${qt(objectWithoutPrototype)},{${property}:`;
-
       // core-js@3 minified pattern.
       // {target:"Array",proto:true},{fill:fill
       // {target:"Array",proto:true,forced:!HAS_SPECIES_SUPPORT||!USES_TO_LENGTH},{filter:
       expression += `|{target:${qt(objectWithoutPrototype)}\\S*},{${property}:`;
     } else {
-      // WeakSet, etc.
-      expression += `|function ${property}\\(`;
+      // Map, Set, WeakSet, etc.
+      // collection("Map",
+      // Currently not used. See create-polyfill-module-data.js
+      // expression += `|collection\\(${qt(property)},`;
```

Comment:
Sorry I looked through `create-polyfill-module-data.js` but I still don't understand why this is unused

Reply:
ye, thanks